import { createError } from '@fastify/error'

export const ChatNotFoundException = createError(
    'ERR_PENDING_CHAT_NOT_FOUND',
    'NÃ£o foi encontrado nenhum chat aberto',
    404,
)
